README.md

```diff
@@ -27,3 +27,16 @@ make test
 cmake -DRPM=centos7 ..
 make package
 ```
+
+### CMake options
+| Option | Default | Definition |
+| ------ | ------ | ---------- |
+| ``CMAKE_INSTALL_PREFIX`` | (Platform dependent) | Where to install each data adapter |
```

Comment:
Double backticks, should they be single?

Reply:
both seem to work, will change them to single.